Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -133,7 +137,10 @@ public class CameraManager {
     ///   - animated: Set to `true` if transition should be animated. `false` by default.
     ///   - duration: Controls the duration of the animation transition. Must be greater than zero if `animated` is true.
     ///   - completion: The completion block to be called after an animated transition. Only called if `animated` is true.
-    public func setCamera(to targetCamera: CameraOptions, animated: Bool = false, duration: TimeInterval = 0, completion: ((UIViewAnimatingPosition) -> Void)? = nil) {
+    public func setCamera(to targetCamera: CameraOptions,
+                          animated: Bool = false,
+                          duration: TimeInterval = 0,
+                          completion: ((UIViewAnimatingPosition) -> Void)? = nil) {
```

Comment:
```suggestion
    public func setCamera(with cameraOptions: CameraOptions,
                          animated: Bool = false,
                          duration: TimeInterval = 0,
                          completion: ((UIViewAnimatingPosition) -> Void)? = nil) {
```

Could we do this instead?

Reply:
I don't feel strongly either way. I think both make sense for the action and are appropriate.

Reply:
@julianrex opinions? `with` feels more like what I'd expect. I think `to` is too strong given that the `CameraOptions` may not be able to be applied exactly.

Reply:
Happy to defer this to a later PR too though

Reply:
@macdrevx lets note and defer on our google doc